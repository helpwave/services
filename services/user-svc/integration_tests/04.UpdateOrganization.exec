#!/bin/bash

echo "Update Organization:"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{ "id":"cc401e43-d76c-4f0e-933a-3ddfab895e59", "long_name": "Bearbeiter-Organisation", "short_name": "Bearbeiter", "contact_email": "edit@helpwave.de", "is_personal": false }' \
	localhost:3002 \
	proto.services.user_svc.v1.OrganizationService/UpdateOrganization

echo "Get Organization:"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "cc401e43-d76c-4f0e-933a-3ddfab895e59"}' \
	localhost:3002 \
	proto.services.user_svc.v1.OrganizationService/GetOrganization 2>&1 || echo ""
