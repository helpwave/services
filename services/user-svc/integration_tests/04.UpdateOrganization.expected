Update Organization:
{}
Get Organization:
{
  "id": "cc401e43-d76c-4f0e-933a-3ddfab895e59",
  "longName": "Bearbeiter-Organisation",
  "shortName": "Bearbeiter",
  "contactEmail": "edit@helpwave.de"
}
